From 00f056bdecb10d1c7059943763f99a55202e72a5 Mon Sep 17 00:00:00 2001
From: "Cardona Ruiz, Erick" <erick.cardona.ruiz@intel.com>
Date: Tue, 29 Mar 2016 11:11:28 -0600
Subject: [PATCH 2/2] Modify rootwrap location

Signed-off-by: Cardona Ruiz, Erick <erick.cardona.ruiz@intel.com>
---
 etc/sahara/rootwrap.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/sahara/rootwrap.conf b/etc/sahara/rootwrap.conf
index 614ce22..8c3aa8e 100644
--- a/etc/sahara/rootwrap.conf
+++ b/etc/sahara/rootwrap.conf
@@ -4,7 +4,7 @@
 [DEFAULT]
 # List of directories to load filter definitions from (separated by ',').
 # These directories MUST all be only writeable by root !
-filters_path=/etc/sahara/rootwrap.d,/usr/share/sahara/rootwrap
+filters_path=/usr/share/nova/rootwrap.d
 
 # List of directories to search executables in, in case filters do not
 # explicitely specify a full path (separated by ',')
-- 
2.5.0

